src/content/docs/apm/transactions/key-transactions/introduction-key-transactions.mdx

```diff
@@ -16,33 +16,230 @@ redirects:
   - /docs/apm/selected-transactions/key-transactions/key-transactions-tracking-important-transactions-or-events
   - /docs/apm/transactions/key-transactions/key-transactions-tracking-important-transactions-or-events
   - /docs/key_transaction/key_transactions
+  - /docs/apm/transactions/key-transactions/create-update-key-transactions
+  - /docs/apm/transactions/key-transactions/tag-key-transactions
 ---
 
-In APM, you can make a transaction a **key transaction** when you want it to have a custom level of monitoring, or when you want it to have its own [alert policy](/docs/apm/transactions/key-transactions/view-key-transactions-alert-information). These may be [web transactions](/docs/using-new-relic/welcome-new-relic/getting-started/glossary#transaction) or [non-web transactions](/docs/using-new-relic/welcome-new-relic/getting-started/glossary#non-web-transaction) that are important to your business.
+import apmKeyTransactionsTrackChanges from 'images/apm_screenshot-full_key-transactions-track-changes.png'
 
-For most New Relic agents, a key transaction can be any type of transaction. For the PHP, Node.js, and Go agents, only **web transactions** can be turned into key transactions.
+import apmServiceMapsandKeyTransactions from 'images/apm_screenshot-full_service-maps-and-key-transactions.png'
 
-## What are key transactions? [#overview]
+import apmLookoutandKeyTransactions from 'images/apm_screenshot-full_lookout-and-key-transactions.png'
 
-In APM, making a [transaction](/docs/apm/transactions/intro-transactions/transactions-new-relic-apm) a **key transaction** gives you the following benefits:
+import apmWorkloadsandKeyTransactions from 'images/apm_screenshot-full_workloads-and-key-transactions.png'
 
-* Key transactions are grouped together in the New Relic UI, so they are easy to find and examine.
-* You can [set a custom Apdex threshold](/docs/apm/transactions/key-transactions/create-key-transactions#apdex) for a key transaction.
-* You can set an [alert policy](/docs/apm/transactions/key-transactions/view-key-transactions-alert-information) for a key transaction.
+Key transactions are a way for engineering teams to create custom level monitoring on their most meaningful transactions. The APM monitoring tools of New Relic help you swiftly detect and solve problems before they impact your users by reporting on _all transactions_. But sometimes the information that is essential to running your digital business isn't the most highly trafficked.  This is where key transactions can help you track essential transactions you want to monitor and manage—regardless of the overall application throughput. 
```

Comment:
It might be nice to mention that if users need help with the concept of transactions and whether their framework supports them, they could see the intro: https://docs.newrelic.com/docs/apm/transactions/intro-transactions/transactions-new-relic-apm/

Reply:
Good idea! This is done!